modules/available-platforms.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/understanding-installations.adoc
+// * architecture/architecture-installation.adoc
+
+[id="available-platforms_{context}"]
+= Available platforms
```

Comment:
I suggest undoing the manual wrapping in these new modules now.

Reply:
Thank you for the suggestion. I have now resolved this for each of the new modules.